horizon/gopal/runner.py

```diff
@@ -31,6 +35,7 @@ def _client(self) -> aiohttp.ClientSession:
         return self.__client
 
     async def __aenter__(self):
+        self.set_envs()
```

Comment:
really don't like the fact we are doing it with env vars.
we probably need to switch to yaml later.

Reply:
we will switch to cobra & viper later on so everything will be available but IMO in the beginning this isn't the most important part